src/server/ua_server.c

```diff
@@ -744,3 +763,19 @@ UA_Server_run(UA_Server *server, const volatile UA_Boolean *running) {
     }
     return UA_Server_run_shutdown(server);
 }
+
+void
+UA_Server_setupEventLoop(UA_Server *server) {
```

Comment:
This is used only in the same file.
Make it a static function to reduce the API surface.